src/Adliance.AspNetCore.Buddy.Pdf/V2/AdliancePdfer.cs

```diff
@@ -42,5 +46,25 @@ public async Task<byte[]> HtmlToPdf(string html, PdfOptions options)
             response.EnsureSuccessStatusCode();
             return await response.Content.ReadAsByteArrayAsync();
         }
+        
+        private int[] ConvertPaperSizeFromMMToPixels(PdfSize? size, int? paperWidth, int? paperHeight)
```

Comment:
Why is this method called "from MM". We don't convert from millimeters here, but use always pixels?